src/overlays/actors/ovl_En_Paper/z_en_paper.c

```diff
@@ -35,114 +39,155 @@ const ActorInit En_Paper_InitVars = {
     (ActorFunc)EnPaper_Draw,
 };
 
-Vec3f D_80C1FC60 = { 0.0f, 0.0f, 1.0f };
+static Vec3f sUnitVectorZ = { 0.0f, 0.0f, 1.0f };
```

Comment:
```suggestion
static Vec3f sUnitVecZ = { 0.0f, 0.0f, 1.0f };
```
Personally I think it is better to shorten it to `Vec`, but I'll leave it up to you.